sonic_platform_base/sonic_xcvr/sfp_optoe_base.py

```diff
@@ -144,3 +144,113 @@ def write_eeprom(self, offset, num_bytes, write_buffer):
         except (OSError, IOError):
             return False
         return True
+
+    def get_lpmode(self):
+        api = self.get_xcvr_api()
+        return api.get_lpmode() if api is not None else False
+
+    def set_lpmode(self, lpmode):
+        api = self.get_xcvr_api()
+        return api.set_lpmode(lpmode) if api is not None else False
+
+    def get_module_state(self):
```

Comment:
new API, please add description.

Reply:
get_module_state() is already merged from someone else in the community